src/Shell/ElasticMappingShell.php

```diff
@@ -63,10 +63,10 @@ public function main($table)
             ],
             'meta' => [
                 'properties' => [
-                    'ip' => ['type' => 'string', 'index' => 'not_analyzed'],
-                    'url' => ['type' => 'string', 'index' => 'not_analyzed'],
-                    'user' => ['type' => 'string', 'index' => 'not_analyzed'],
-                    'app_name' => ['type' => 'string', 'index' => 'not_analyzed']
+                    'ip' => ['type' => 'text', 'index' => 'not_analyzed'],
+                    'url' => ['type' => 'text', 'index' => 'not_analyzed'],
+                    'user' => ['type' => 'text', 'index' => 'not_analyzed'],
+                    'app_name' => ['type' => 'text', 'index' => 'not_analyzed']
```

Comment:
These are not the only changes this shell needs. You also need to change the logic that creates the index.

Before the index name was taken out of the config name. Now it should be taken maybe from the type? This is not too clear so far

Reply:
I agree.

Into ElasticSearchPersister, I chose to store the name from constructor parameter, not automagically getting it - AuditLogBehavior define it when initializing.

For creating mapping and index, maybe we can get that name also from Behavior / Persister.

Reading some docs from Elasticsearch, I realized is usual to name 'index' in plural and 'type' in singular (just as we do in ORM for table -> entity). I will do the same when infering 'type' from 'index' name.